gradle/project.gradle.kts

```diff
@@ -75,6 +75,12 @@ allprojects {
 
     tasks.withType<KotlinCompile> {
         kotlinOptions.jvmTarget = "1.8"
+        kotlinOptions.jdkHome = dirJdk.path
+        kotlinOptions.apiVersion = "1.3"
+        kotlinOptions.languageVersion = "1.3"
```

Comment:
Are you sure? My intention was to be explicit and prevent possible problem where IntellijIdea would override the settings. I was actually experiencing a problem where every time build scrip was rebuilt, Intellj would change jvmTarget for all modules to 1.6. Setting it manually to 1.8 in Idea only had effect until next build. I think the issue stopped after trying like everything (these changes included). I can remove it as I understand the potential overhead, but I think the other side of the coin is potentially more troubling. And api version/languageVersion is not going to change for like the next year, plus, developer updating the Kotlin should actually be aware of such settings instead of blindly counting on sensible default. No?

Reply:
jvmTarget should definitely be set, idea was probably pulling its setting from Gradle, and the default is 1.6. However, the api and languageVersion are always the latest by default. 
Updating the plugin should be enough to reach the developers awareness ^^

Reply:
Ok, will remove.